### PR TITLE
Fix tanh typo for rvv.

### PR DESCRIPTION
**Pull request by zhangyang2057**

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=5584) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=5584) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/ncnn?pullRequest=5584) it.</sub>

File: src/layer/riscv/rvv_mathfun.h [hunk 1]
```diff
@@ -308,7 +308,7 @@ _RVV_FLOAT32_COS_OP(8, 4)
                                                                                                  \
         /* clamp the inputs to the range [-9, 9] since anything outside */                       \
         /* this range is -/+1.0f in single-precision.                   */                       \
-        x2 = vfmin_vf_f32m##LMUL(x, c_tanh_hi, vl);                                              \
+        x2 = vfmin_vf_f32m##LMUL(x2, c_tanh_hi, vl);                                             \
```
Comment (nihui) [inline src/layer/riscv/rvv_mathfun.h:311]: and the same fix in rvv_mathfun_fp16s.h ?

Comment (zhangyang2057) [reply src/layer/riscv/rvv_mathfun.h:311]: Yes, ditto.

**Review by nihui — APPROVED** [review]

**Comment by nihui** [comment]
Thanks for your contribution !

**Pull request merged**